app/domains/schulden/vorabcheck/xstateConfig.ts

```diff
@@ -117,5 +119,86 @@ export const kontopfaendungWegweiserXstateConfig = {
         },
       },
     },
+    zwischenseite: {
+      initial: "unterhalt",
+      states: {
+        unterhalt: {
+          on: {
+            SUBMIT: "#/schulden/kontopfaendung/wegweiser.unterhalt",
+            BACK: "#/schulden/kontopfaendung/wegweiser.abfrage-basis-infos.euro-schwelle",
+          },
+        },
+      },
+    },
+    unterhalt: {
+      initial: "kinder",
+      states: {
+        kinder: {
+          on: {
+            SUBMIT: [
+              {
+                target: "natural",
+                guard: ({ context }) =>
+                  context.hasKinder === YesNoAnswer.Values.yes,
+              },
+              {
+                target: "partner",
+                guard: ({ context }) =>
+                  context.hasKinder === YesNoAnswer.Values.no,
+              },
+            ],
+            BACK: "#/schulden/kontopfaendung/wegweiser.zwischenseite.unterhalt",
+          },
+        },
+        natural: {
+          on: {
+            SUBMIT: [
+              {
+                target: "kinder-support",
+              },
+            ],
+            BACK: "kinder",
+          },
+        },
+        "kinder-support": {
+          on: {
+            SUBMIT: [
+              {
+                target: "partner",
+              },
+            ],
+          },
+        },
+        partner: {
+          on: {
+            SUBMIT: [
+              {
+                target: "zusammenwohnen",
+                guard: ({ context }) =>
+                  context.verheiratet !== verheiratetType.Values.nein,
+              },
+              {
+                target: "partner-support",
+                guard: ({ context }) =>
+                  context.verheiratet === verheiratetType.Values.nein,
+              },
```

Comment:
could reverse the condition here and make the logic easier like so:
```suggestion
             {
                target: "partner-support",
                guard: ({ context }) =>
                  context.verheiratet === verheiratetType.Values.nein,
              },
              "zusammenwohnen",
```

Reply:
also the target for `verheiratet === nein` should be blank for now I think, as it looks like it's targeting the next zwischenseite